### PR TITLE
sets up web scraping and databases

### PR DESCRIPTION
**Pull request by ZombiMigz**

### CODE REVIEW

File: src/models.py [hunk 1]
```diff
@@ -0,0 +1,9 @@
+from django.db import models
```
Comment (ZombiMigz) [inline src/models.py:1]: This is the actual schema, I realize running multiple scrapes will duplicate the data but that's a problem for later.

File: src/migrations/0001_initial.py [hunk 2]
```diff
@@ -0,0 +1,25 @@
+# Generated by Django 5.0.4 on 2024-05-03 07:27
```
Comment (ZombiMigz) [inline src/migrations/0001_initial.py:1]: These migrations are auto generated by django as a record for when the schema changes, I was told to commit them so 🤷, I assume no one will need another migration either way

File: .vscode/settings.json [hunk 3]
```diff
@@ -1,3 +1,7 @@
 {
-  "python.analysis.typeCheckingMode": "strict"
+  "python.analysis.typeCheckingMode": "strict",
+  "python.analysis.autoImportCompletions": true,
+  "python.analysis.diagnosticSeverityOverrides": {
+    "reportUnknownMemberType": "none"
```
Comment (ZombiMigz) [inline .vscode/settings.json:5]: had to add this one to appease pylance for unknown types defined by selenium

File: src/management/commands/scrape.py [hunk 4]
```diff
@@ -0,0 +1,104 @@
...
+        for day_of_the_week in days:
+            self.insert_section_into_database(
+                building, room, start_time, end_time, day_of_the_week
+            )
+
+    def handle(self, *args: Tuple[Any], **kwargs: dict[str, Any]):
+        print("Starting Scrape")
+
+        # Start up browser and go to schedule website
+        driver = webdriver.Chrome()
```
Comment (ZombiMigz) [inline src/management/commands/scrape.py:71]: lmk if this doesn't work on your pcs, I already had some selenium driver stuff setup globally so not sure if it'd work on a clean install

**Review by phillip-che — APPROVED** [review]
looks good

**Pull request merged**